Library/Formula/i2p.rb

```diff
@@ -0,0 +1,24 @@
+class I2p < Formula
+  desc "I2P is an anonymous overlay network - a network within a network"
+  homepage "https://geti2p.net"
+  url "https://download.i2p2.de/releases/0.9.21/i2pinstall_0.9.21.jar"
+  sha256 "0238ffc6ea44099ef4fe6c20913cb4eec675c2760aea07dfe7d499addcc89cf2"
+
+  depends_on :java => "1.6+"
+
+  def install
+    (buildpath/"path.conf").write "INSTALL_PATH=#{libexec}"
+    system "java", "-jar", "i2pinstall_#{version}.jar", "-options", "path.conf"
+    wrapper_name = "i2psvc-macosx-universal-#{MacOS.prefer_64_bit? ? '64' : '32'}"
+    libexec.install_symlink libexec/wrapper_name => "i2psvc"
+    bin.write_exec_script Dir["#{libexec}/{eepget,i2prouter}"]
+    man1.install Dir["#{libexec}/man/*"]
+  end
+
+  test do
+    system bin/"i2prouter", "start"
+    sleep 5
+    assert_match(/I2P Service is running/, shell_output("#{bin}/i2prouter status"))
+    system bin/"i2prouter", "stop"
```

Comment:
Let's try and use something like [this](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/mpd.rb#L127-L139). At the moment with the current test if there's a problem on the `assert_match` it'll bail out and leave `i2prouter` running which isn't desirable.